src/anemoi/training/diagnostics/maps.py

```diff
@@ -32,7 +32,7 @@ def __init__(self) -> None:
     def __call__(self, lon: np.ndarray, lat: np.ndarray) -> tuple[np.ndarray, np.ndarray]:
         lon_rad = np.radians(lon)
         lat_rad = np.radians(lat)
-        x = [v - 2 * np.pi if v > np.pi else v for v in lon_rad]
+        x = np.array([v - 2 * np.pi if v > np.pi else v for v in lon_rad], dtype=np.float32)
```

Comment:
Why is this being converted to a numpy array? 
Is this another migrated change?

Reply:
Yes this is a fix from the PR already opened in aifs-mono. This is basically to ensure that they function type hint is the correct otherwise x would be a list not an array. We also realised that ensuring both x and y are arrays does have a impact on the matplotlib or in general in the plotting generation.